android/src/main/kotlin/com/apptreesoftware/barcodescan/BarcodeScannerActivity.kt

```diff
@@ -160,6 +160,18 @@ class BarcodeScannerActivity : Activity(), ZXingScannerView.ResultHandler {
                 window.addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS)
                 window.statusBarColor = 0xFF2A1052.toInt()
             }
+        } else if (theme != null && theme.equals("obsidium")) {
+            actionBarColor = 0xFF5C9BE5.toInt()
+            scannerView.setBorderColor(0xFF5C9BE5.toInt())
+            actionBar?.setBackgroundDrawable(ColorDrawable(0xFF1C1CE.toInt()))
```

Comment:
Must have made a typo, this hex code is invalid which is why the barcode scanner view has that ugly white/action bar

Reply:
Typo should be fixed in the latest commit.